test/integration/server-selection/server_selection.prose.operation_count.test.ts

```diff
@@ -163,8 +163,8 @@ describe('operationCount-based Selection Within Latency Window - Prose Test', fu
     const [host1, host2] = seeds.map(seed => seed.split(':')[1]);
     const percentageToHost1 = (counts[host1] / totalNumberOfTasks) * 100;
     const percentageToHost2 = (counts[host2] / totalNumberOfTasks) * 100;
-    expect(percentageToHost1).to.be.greaterThan(40).and.lessThan(60);
-    expect(percentageToHost2).to.be.greaterThan(40).and.lessThan(60);
+    expect(percentageToHost1).to.be.greaterThan(35).and.lessThan(65);
+    expect(percentageToHost2).to.be.greaterThan(35).and.lessThan(65);
```

Comment:
Were we getting any flakiness on the lower bound as well?

Reply:
We have in the past, although not recently.  

Keep in mind that this test monitors what percentage of operations get sent to each mongos in a sharded cluster with two mongoses.  So if we go above 60% in one, we drop below 40% in the other.  So we need to update the lower bound too.